ACMAS/app/ACMAS/settings.py

```diff
@@ -39,6 +39,8 @@
     "django.contrib.messages",
     "django.contrib.staticfiles",
     "ACMAS_Web",
+    "compressor",
```

Comment:
I don't think we're actually using this for our tailwindcss implementation, so I'll make a commit soon to remove it.